NFIQ2/NFIQ2Algorithm/src/features/FeatureFunctions.cpp

```diff
@@ -391,6 +391,20 @@ void getRidgeValleyStructure( const Mat& blockCropped, std::vector<uint8_t>& rid
     throw NFIQ::NFIQException( NFIQ::e_Error_FeatureCalculationError, ssErr.str() );
   }
 
+  // Round to 11 decimal points to preserve score consistency across platforms (10^11)
+  std::function<double( double )> truncateEleven = [&]( double val )
+  {
+    return round( val * 100000000000 ) / 100000000000;
+  };
+
+  for( int i = 0; i < dt1.rows; i++ )
```

Comment:
@afiglarz can you try to use `Mat`'s [`forEach()`](https://docs.opencv.org/master/d3/d63/classcv_1_1Mat.html#afcee31e71fd6991148fd3bb619971e2b) method instead?